.github/actions/spell-check/patterns.txt

```diff
@@ -12,11 +12,24 @@ https?://(?:(?:www\.|)youtube\.com|youtu.be)/[-a-zA-Z0-9?&=]*
 # msdn
 \b(?:download\.visualstudio|docs|msdn)\.microsoft\.com/[-_a-zA-Z0-9()=./]*
 
-data:[a-zA-Z=;,/0-9+-]+
-0x[0-9a-fA-F](?:\.[0-9a-fA-F]*|)[pP]
-(?:0[Xx]|U\+|#)[a-f0-9A-FGgRr]{2,}[Uu]?[Ll]{0,2}\b
+publicKeyToken=(['"]|)[0-9a-f]+\g{-1}
+\@sha256:[0-9a-f]{64}\b
+
+# data urls
+(['"])data:.*?\g{-1}
+data:[-a-zA-Z=;:/0-9+]*,\S*
+
 # uuid:
-[-<({"'>][0-9a-fA-F]{8}-(?:[0-9a-fA-F]{4}-){3}[0-9a-fA-F]{12}[<'"})>]
+L?(["']|[-<({>]|\b)[0-9a-fA-F]{8}-(?:[0-9a-fA-F]{4}-){3}[0-9a-fA-F]{12}(?:\g{-1}|[<})>])
```

Comment:
This is the core change to try to get the spell checker to properly catch a couple of other edges of the GUID family. I can't quite explain all the ins and outs here.

Normally if there's trouble w/ the boundaries, I just relax the prefix from `[<({"'>]` to `\b` -- but that didn't seem to work particularly well.